### ISSUE TITLE
6 sockets

### ISSUE THREAD

**Issue by lsmllc** [notablackbear/poe_qolV2#50]
6 sockets lvl 60+ normal/magic/rare chests are counted as Body for chaos recipe.
![6s](https://user-images.githubusercontent.com/10700834/93966060-bdeef380-fd31-11ea-8155-be59cb9dbb4c.png)

**Comment by notablackbear** [comment]
Sorry, I don't understand the problem. Could you provide some more description? It will treat a 6 socket item the same as any other rare. Does this disrupt the Chaos Recipe because of the 7-Jewellers recipe?

**Comment by lsmllc** [comment]
> Sorry, I don't understand the problem. Could you provide some more description? It will treat a 6 socket item the same as any other rare. Does this disrupt the Chaos Recipe because of the 7-Jewellers recipe?

Exactly, the small overlay count them as if they were chaos recipe body.

**Comment by detemegandy** [comment]
solution: keep a separate dump tab or just sell them
